README.md

```diff
@@ -2,20 +2,33 @@
+### Streamlining Workflows
+First and foremost, this shift calls for a more streamlined and intuitive printing experience. Developers can design print workflows that automate repetitive tasks, preconfigure print settings with respect to a particular printer's capabilities, and eliminate unnecessary user interactions.
 
-### Leveraging Progressive Web Apps
-The new Web API can motivate developers to switch to Progressive Web Apps (PWAs) by offering enhanced printing capabilities that were traditionally limited to native applications (in particular, support for Printer Redirection).
+### Knowing Your Printers
+To enable customization of settings, developers need to have knowledge about the capabilities of the printer. The API provides a convenient method for accessing this information, which enhances the printing experience by allowing developers to retrieve relevant details about what the printer can and cannot do. A few examples:
+* If the printer supports high-resolution printing, the application can automatically select the appropriate print resolution, resulting in sharper and more detailed output.
+* If the printer supports duplex printing, the application can prompt the user to print double-sided, saving paper and reducing printing time.
+* Access to supported media sizes empowers the application to generate print-ready files by adjusting page layouts, margins, or scaling, effectively ensuring that the content fits properly on the printed page without any cropping or distortion.
+
+### Observing the Printing Pipeline
+When users utilize `window.print()`, they have limited visibility and control over the printing process - usually it's scoped to hidden system UI surfaces & system notifications that cannot be accessed by the app. The API strives to fill this gap by offering the ability to track the progress of print jobs as well as cancel them if needed directly to the applications, creating a more evident and clear printing experience via custom information surfaces.
+
+### Improving Error Handling
+Similarly to the previous point, there are no built-in error handling mechanisms -- this effectively leaves the user alone against the print system when someting goes wrong during printing. With more control over the printing process, apps can wrap up raw printing errors into more elaborate descriptions (what went wrong and why?) or suggest useful tips for troubleshooting common errors (potentially even going as far as providing links to vendor websites or user manuals for specific printers since model info is also available now).
+
+### Supporting Remote Printing
+The proposed API methods facilitate printer forwarding by enabling the remote client to access essential information about printers on the near side: this allows the remote client to interact with and control those printers as if they were directly connected to the remote client itself, enhancing the seamless integration and operation of printers in the remote environment.
```

Comment:
A key part to this is that generating the print job remotely **requires** knowledge of the selected printer's capabilities, as otherwise the remote system could, for example, generate a document formatted for A4 paper when the local printer uses US Letter-sized paper. It's not just that this makes the integration more seamless, without tedious manual configuration this feature simply won't work.